<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Numbers_memory</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(145, 26, 143, 150), stop:1 rgba(23, 30, 133, 125));</string>
  </property>
  <property name="iconSize">
   <size>
    <width>80</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonStart">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>150</y>
      <width>230</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border-color: rgb(92, 0, 149);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="change_complexity">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Выберете сложность:</string>
    </property>
   </widget>
   <widget class="QWidget" name="Levels" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>366</y>
      <width>90</width>
      <height>95</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="complexity_light">
       <property name="styleSheet">
        <string notr="true">background-color: none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>легко</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="complexity_middle">
       <property name="styleSheet">
        <string notr="true">background-color: none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>средне</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="complexity_hard">
       <property name="styleSheet">
        <string notr="true">background-color: none;
font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>сложно</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
